test/e2e/llmisvc/test_llm_tls.py

```
@@ -0,0 +1,224 @@
+        logger.error(f"Failed TLS verification for {service_name}: {e}")
+        _collect_diagnostics(kserve_client, test_case.llm_service)
+        raise
+    finally:
+        try:
+            if os.getenv("SKIP_RESOURCE_DELETION", "False").lower() in (
+                "false",
+                "0",
+                "f",
+            ):
+                delete_llmisvc(kserve_client, test_case.llm_service)
+        except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)